Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -31,14 +31,21 @@ DECLARE_SOA_STORE();
 
 namespace bc
 {
-DECLARE_SOA_COLUMN(RunNumber, runNumber, int);          //! Run number
-DECLARE_SOA_COLUMN(GlobalBC, globalBC, uint64_t);       //! Bunch crossing number (globally unique in this run)
-DECLARE_SOA_COLUMN(TriggerMask, triggerMask, uint64_t); //! CTP trigger mask
+DECLARE_SOA_COLUMN(RunNumber, runNumber, int);              //! Run number
+DECLARE_SOA_COLUMN(GlobalBC, globalBC, uint64_t);           //! Bunch crossing number (globally unique in this run)
+DECLARE_SOA_COLUMN(TriggerMask, triggerMask, uint64_t);     //! CTP trigger mask
+DECLARE_SOA_COLUMN(TriggerInputs, triggerInputs, uint64_t); //! CTP inputs
 } // namespace bc
 
-DECLARE_SOA_TABLE(BCs, "AOD", "BC", o2::soa::Index<>, //! Root of data model for tables pointing to a bunch crossing
+DECLARE_SOA_TABLE(BCs_000, "AOD", "BC", o2::soa::Index<>, //! Root of data model for tables pointing to a bunch crossing, version 000
                   bc::RunNumber, bc::GlobalBC,
                   bc::TriggerMask);
+
+DECLARE_SOA_TABLE_VERSIONED(BCs_001, "AOD", "BC", 1, o2::soa::Index<>, //! Root of data model for tables pointing to a bunch crossing, version 001
+                            bc::RunNumber, bc::GlobalBC,
+                            bc::TriggerMask, bc::TriggerInputs);
+
+using BCs = BCs_001; //! this defines the current default version
```

Comment:
Maybe we don't need to split any PR: @shahor02, can we just change this line to say `using BCs = BCs_000;` and merge? This will be a fully transparent change and then I will do the O2Physics PR for the converter. Changing one line to `using BCs = BCs_001` later, once the converter is in O2Physics.

Reply:
But then the aod producer does not compile...

Reply:
It is different because one additional member is added here. In your PR, the members are just swapped.

Reply:
Can you send me the error you get on mattermost?

Reply:
@shahor02, everyone, how about we just roll back the AO2D producers ([like this](https://github.com/shahor02/AliceO2/pull/34)) until we get the converter in place? Thanks!